### PR TITLE
Add new placement mode for displacement renderer

### LINKED ISSUE #642: Add concentric ring placement mode for displacement renderer
https://github.com/qgis/QGIS/commit/73d838fd810fba419a0105e87c78b6b6b2087810
Author: Nyall Dawson
Date:   Fri Sep 18 13:20:53 2015 +1000

### PR DESCRIPTION
**Pull request by yjacolin**
* Add concentric ring placement mode for displacement renderer
* Move tips at the top of the section

Fix #642

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -580,6 +580,20 @@ symbolizing your vector data.
...
+
+   You have the option to export vector symbology from QGIS into Google \*.kml,
+   \*.dxf and MapInfo \*.tab files. Just open the right mouse menu of the layer
+   and click on :menuselection:`Save As...` to specify the name
+   of the output file and its format.
+   In the dialog, use the :menuselection:`Symbology export` menu to save the symbology
+   either as :menuselection:`Feature symbology -->` or as :menuselection:`Symbol
+   layer symbology -->`.
+   If you have used symbol layers, it is recommended to use the second setting.
+
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Though I wonder why only these three formats were listed, could ["Save and share properties"](http://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/vector_properties.html#save-and-share-layer-properties) or ["Layers Panel"](http://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_gui.html#layers-panel) section be a better candidate for this block?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -918,18 +932,9 @@ placed on a displacement circle around a center symbol.
...
-   of the output file and its format.
-   In the dialog, use the :menuselection:`Symbology export` menu to save the symbology
-   either as :menuselection:`Feature symbology -->` or as :menuselection:`Symbol
-   layer symbology -->`.
-   If you have used symbol layers, it is recommended to use the second setting.
-
-.. ToDo: add information about the export options
+.. note:: You can still render features with other renderer like Graduaded,
+   Categorized, or Rule-Based renderer using the `Renderer` dropdown list then
+   the `Renderer Settings ...` button.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: typos: Gradua**t**ed and drop**-**down

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Weel I just move this part in other "better" place but I wondered the same thing. Could we create another issue for this and move it later somewhere else for 2.16?

**Comment by yjacolin** [comment]
Just lost this PR when rebasing. Too much changes in this files, nobody can resolve this kind of conflicts. :/

**Comment by yjacolin** [comment]
Note sure if I retrieved all my past commits. Most of it probably.

**Pull request merged**